.github/workflows/ci.yml

```diff
@@ -107,6 +107,12 @@ jobs:
   docker:
     needs: [test]
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        model-name:
+          - "dslim/bert-base-NER"
```

Comment:
You can not have `/` in the model names, otherwise `IMAGE=${{ env.docker_repository_base }}-${{ matrix.model-name }}` below will not be a valid docker image name.